docs/spark/tutorials/databricks-deployment.md

```diff
@@ -125,14 +125,14 @@ Next, you publish the *mySparkApp* created in the [.NET for Apache Spark - Get S
 
    **On Windows:**
 
-   ```console
+   ```dotnetcli
    cd mySparkApp
    dotnet publish -c Release -f netcoreapp3.0 -r ubuntu.16.04-x64
    ```
 
    **On Linux:**
 
-   ```bash
+   ```dotnetcli
    cd mySparkApp
    dotnet publish -c Release -f netcoreapp3.0 -r ubuntu.16.04-x64
    ```
```

Comment:
I noticed that commands are exactly the same. Either there is a mistake and they should be different, or they should be combined together.

Reply:
@mamccrea Can you comment on this?

Reply:
@mamccrea ping

Reply:
Yes, they are exactly the same and one should be removed. I think the original thought was the difference in "bash" for Linux vs. "console" for windows, but they're the same commands